environment.yml

```diff
@@ -6,13 +6,14 @@ dependencies:
 - hdf5
 - netcdf4
 - numpy
-- pytables
 - pandas
-- xarray
-- scikit-learn
-- scipy
-- ipython
-- matplotlib
+- pip
+- pyclipper
 - pyproj
 - python=3.6
-- six
\ No newline at end of file
+- scipy
+- six
```

Comment:
Is `six` needed? This is a python 3-only package, right?

Reply:
More importantly, it's not a dependency of this code base, so IMO it shouldn't be listed. Let the other packages provide their dependencies.

Reply:
Just pushed a commit to remove six, as well as pyproj, per the principles articulated above.